apps/task-app/src/app/containers/details/styled-components.tsx

```diff
@@ -38,23 +38,27 @@ export const ListWithDetailHeading = styled.h3`
 export const ReviewContainer = styled.div`
   display: flex;
   flex-direction: column;
+  align-items:center;
+  padding-top: 24px !important;
```

Comment:
can you use design tokens for the 24px
use: var(--goa-space-l);

Reply:
@thdlam  updated the PR